changelog.txt

```diff
@@ -1,3 +1,7 @@
+0.78
+ * Fixed bug causing Realms to be cached during a RealmMigration, resulting in invalid realms being returned.
```

Comment:
The description is better in the description of this PR :-)
e.g. "realms being returned" - from where?